h2o-genmodel/src/main/java/hex/genmodel/easy/EasyPredictModelWrapper.java

```diff
@@ -599,18 +599,26 @@ public BinomialModelPrediction predictBinomial(RowData data, double offset) thro
     return p;
   }
 
+  /**
+   * @deprecated Use {@link #predictTargetEncoding(RowData)} instead.
+   */
+  @Deprecated
+  public TargetEncoderPrediction transformWithTargetEncoding(RowData data) throws PredictException{
+    return predictTargetEncoding(data);
+  }
+
   /**
    * Perform target encoding based on TargetEncoderMojoModel
    * @param data RowData structure with data for which we want to produce transformations
    * @return TargetEncoderPrediction with transformations ordered in accordance with corresponding categorical columns' indices in training data
    * @throws PredictException
    */
-  public TargetEncoderPrediction transformWithTargetEncoding(RowData data) throws PredictException{
+  public TargetEncoderPrediction predictTargetEncoding(RowData data) throws PredictException{
```

Comment:
well... this is what I suggested originally but it had to be transform back then...

Reply:
Yeah, I think that at API level, consistency is more important than perfect naming... and word2vec has a similar semantic as TE but it is still exposed as `predictWord2Vec`, so I don't see any reason to keep (outside backwards compatibility) this `transform__`.

Reply:
Internal conventions aside, I do think that "transform" is the more sensible verb to describe what we're doing here (it's not really predicting anything in the ML sense of the word "predict"; it's just computing the values).  It sounds like the preference to use the "predict" verb comes with the desire to be treating TE as an "estimator", or is there something more than that?  Are there other methods like "predictSomething" in the Python API?

Reply:
I think `predict` is currently better simply by convention: I agree that semantically it's not the perfect choice for TargetEncoder, Word2Vec, AutoEncoder, DimReduction...

For the future, maybe we could keep this `predict` prefix everywhere for consistency, and for clarity, add a `transform` or `apply` alias for those who actually don't produce predictions.